FAQ.md

```diff
@@ -39,9 +39,7 @@ Intel processors. AMD and ARM processors will be supported in the near future.
 
 ### Can Firecracker be used with Kubernetes, Docker, or Kata containers today?
 
-Not yet. We are making Firecracker open source because it provides a
-meaningfully different approach to security for running containers. We hope that
-others in the communities that build open source container technology find it
+Not yet with Kubernetes and Docker, but since the last [1.5.0](https://github.com/kata-containers/runtime/releases/tag/1.5.0) release of Kata Containers, support for the Firecracker hypervisor has been included. We hope that others in the communities that build open source container technology find it
```

Comment:
Remove the word `last` from `last release` since that will only be true for a limited amount of time.

Reply:
Thanks for your suggestion. Done.